enforcer-rules/README.md

```diff
@@ -0,0 +1,94 @@
+# Linkage Checker Enforcer Rule
+
+This Maven enforcer rule verifies that the transitive dependency tree of `pom.xml` does not have
+[linkage error](../library-best-practices/glossary.md#types-of-conflicts-and-compatibility).
+
+## Class path and BOM flag
+
+The rule takes `bom` flag to control the behavior (by default `false`).
```

Comment:
Is `bom` the right word here? A parent POM such as https://github.com/GoogleCloudPlatform/cloud-opensource-java/blob/master/pom.xml also has a dependencyManagement section that I think this works with.